src/amo/components/AddonMeta.js

```diff
@@ -8,24 +8,54 @@ import 'amo/css/AddonMeta.scss';
 
 export class AddonMetaBase extends React.Component {
   static propTypes = {
-    averageDailyUsers: PropTypes.number.isRequired,
+    addon: PropTypes.object.isRequired,
     i18n: PropTypes.object.isRequired,
   }
 
   render() {
-    const { averageDailyUsers, i18n } = this.props;
+    const { addon, i18n } = this.props;
+    const averageDailyUsers = addon.average_daily_users;
+    const averageRating = addon.ratings.average;
+    const addonReviewCount = addon.ratings.count;
 
     const userCount = i18n.sprintf(
       i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
       { total: i18n.formatNumber(averageDailyUsers) },
     );
+    let starCount;
+    if (averageRating) {
+      starCount = i18n.sprintf(
+        i18n.gettext('%(averageRating)s out of 5'),
+        { averageRating: i18n.formatNumber(averageRating) },
+      );
+    } else {
+      starCount = i18n.gettext('Not yet rated');
```

Comment:
I think there are pros and cons to each one but I don't feel too strongly. I think the best thing to do is just land it and get feedback from UX. We had requested feedback on this anyway and hadn't received it yet so other things about it might change.